templates/project.html

```diff
@@ -353,14 +370,17 @@ <h3 class="question-group-title">{{group.title}}</h3>
                         {% if layout_mode != "grid" and not q.can_start_new_task %}
                           {% with task=q.tasks.0 %}
                             <div class="progress" style="width:80%; height: 5px;margin: 8px 0 0 0;">
-                              <div class="progress-bar progress-bar-success" role="progressbar" style="width: {{q.progress_percent}}%" aria-valuenow="{{q.progress_percent}}" aria-valuemin="0" aria-valuemax="100"></div>   
+                              <div class="progress-bar progress-bar-success" role="progressbar" style="width: {{q.progress_percent}}%" aria-valuenow="{{q.progress_percent}}" aria-valuemin="0" aria-valuemax="100"></div>
```

Comment:
This is meant to be deleted since you duplicated it in the if block below to only show progress if the question has been started but is not finished?

Reply:
Ok. I think I've cleaned this up.